### ISSUE TITLE
qwen3-vl-8b训练

### ISSUE THREAD

**Issue by SupreCyk** [verl-project/verl#4193]
### System Info

<img width="1649" height="1042" alt="Image" src="https://github.com/user-attachments/assets/76072785-bec7-4e6a-8f45-15e577a3aa08" />同样的训练配置，但是两个模型的表现效果差异很大，qwen3-vl-8b会经常波动，用的verl仓库代码是10月初的，不知道是不是框架的原因

### Information

- [ ] The official example scripts
- [x] My own modified scripts

### Tasks

- [x] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

nope

### Expected behavior

准备更新一下框架代码试试

**Comment by GmailF** [comment]
> ### System Info
> <img alt="Image" width="822" height="519.4140625" src="https://private-user-images.githubusercontent.com/81468645/516252670-76072785-bec7-4e6a-8f45-15e577a3aa08.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjM2OTUyODQsIm5iZiI6MTc2MzY5NDk4NCwicGF0aCI6Ii84MTQ2ODY0NS81MTYyNTI2NzAtNzYwNzI3ODUtYmVjNy00ZTZhLThmNDUtMTVlNTc3YTNhYTA4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTExMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMTIxVDAzMTYyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlmYjBiYjBmODhlYjk2OTgxYWRlZDNmMzQzZmE2MzhjMDRjYzY4Nzg0YTlkZWNiZWQxMWE4NDQ2ZmM4MmY4ZmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ru9QoKOZgz9uJTI_QMT7MomFsVB1F6V2RCvwafa-nLQ">同样的训练配置，但是两个模型的表现效果差异很大，qwen3-vl-8b会经常波动，用的verl仓库代码是10月初的，不知道是不是框架的原因
> 
> ### Information
> * [ ]  The official example scripts[x]  My own modified scripts
> 
> ### Tasks
> * [x]  An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)[ ]  My own task or dataset (give details below)
> 
> ### Reproduction
> nope
> 
> ### Expected behavior
> 准备更新一下框架代码试试

请教大佬，现在的版本支持 qwen3-vl 架构了吗？我用的版本可能有点旧，不支持

**Comment by SupreCyk** [comment]
#

> > ### System Info
> > <img alt="Image" width="822" height="519.4140625" src="https://private-user-images.githubusercontent.com/81468645/516252670-76072785-bec7-4e6a-8f45-15e577a3aa08.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjM2OTUyODQsIm5iZiI6MTc2MzY5NDk4NCwicGF0aCI6Ii84MTQ2ODY0NS81MTYyNTI2NzAtNzYwNzI3ODUtYmVjNy00ZTZhLThmNDUtMTVlNTc3YTNhYTA4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTExMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMTIxVDAzMTYyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlmYjBiYjBmODhlYjk2OTgxYWRlZDNmMzQzZmE2MzhjMDRjYzY4Nzg0YTlkZWNiZWQxMWE4NDQ2ZmM4MmY4ZmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ru9QoKOZgz9uJTI_QMT7MomFsVB1F6V2RCvwafa-nLQ">同样的训练配置，但是两个模型的表现效果差异很大，qwen3-vl-8b会经常波动，用的verl仓库代码是10月初的，不知道是不是框架的原因
> > ### Information
> > 
> > 
> > * [ ]   The official example scripts[x]  My own modified scripts
> > ### Tasks
> > 
> > 
> > * [x]   An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)[ ]  My own task or dataset (give details below)
> > ### Reproduction
> > nope
> > ### Expected behavior
> > 准备更新一下框架代码试试
> 
> 请教大佬，现在的版本支持 qwen3-vl 架构了吗？我用的版本可能有点旧，不支持

我现在verl的版本是分支V0.6.1,照着安装教程重走了一遍，可以运行，但是qwen3-vl-4b和8b还是会这样一直保持高位抖动

**Comment by bug-to-share** [comment]
请问现在解决了吗，是什么问题呀

**Comment by SupreCyk** [comment]
> 请问现在解决了吗，是什么问题呀

我不知道是不是现在还没支持fsdp的训练方式，我是直接拿的https://github.com/volcengine/verl/blob/main/examples/grpo_trainer/run_qwen2_5_vl-7b.sh 换了下模型然后跑的

**Comment by z-x-x136** [comment]
为什么你的grad_norm那么小，我的100-300

**Comment by Kirrito-k423** [comment]
感谢反馈。当前 main 分支已经提供 Qwen3-VL-8B 的官方 GRPO 示例脚本：examples/grpo_trainer/run_qwen3_vl_8b_fsdp.sh 和 examples/grpo_trainer/run_qwen3_vl_8b_megatron.sh，README 的模型矩阵也列出 Qwen3-VL-8B 支持 FSDP/Megatron。你这里使用的是 v0.6.1，并且是把 Qwen2.5-VL 脚本替换成 Qwen3-VL 模型后运行；从现有信息看，还不足以判断高位抖动是框架 bug、环境版本问题、超参/数据问题，还是 Qwen3-VL 训练稳定性问题。

为了继续定位，请补充：1) 完整启动命令和所有 Hydra override；2) verl commit、transformers/vllm/sglang/torch 版本；3) GPU/NPU 型号、卡数、FSDP 配置；4) wandb/console 中 reward、entropy、kl、grad_norm、clip_ratio 的曲线或关键日志；5) 是否能用当前 main 的 examples/grpo_trainer/run_qwen3_vl_8b_fsdp.sh 在 Geo3K 或最小数据上复现。

Thanks for the report. The current main branch has official Qwen3-VL-8B GRPO example scripts: examples/grpo_trainer/run_qwen3_vl_8b_fsdp.sh and examples/grpo_trainer/run_qwen3_vl_8b_megatron.sh, and the README support matrix lists Qwen3-VL-8B with FSDP/Megatron support. In this issue, the run uses v0.6.1 and a Qwen2.5-VL script with only the model swapped, so the available information is not enough to tell whether the oscillation is caused by a framework bug, dependency/version mismatch, hyperparameters/data, or Qwen3-VL training instability.

Please share: 1) the full launch command and Hydra overrides; 2) verl commit plus transformers/vllm/sglang/torch versions; 3) hardware and FSDP settings; 4) reward/entropy/kl/grad_norm/clip_ratio logs or curves; 5) whether current main's examples/grpo_trainer/run_qwen3_vl_8b_fsdp.sh reproduces the issue on Geo3K or a minimal dataset.

This response was generated by a scheduled Codex GPT5.5 issue processing task.

**Comment by SupreCyk** [comment]
> 感谢反馈。当前 main 分支已经提供 Qwen3-VL-8B 的官方 GRPO 示例脚本：examples/grpo_trainer/run_qwen3_vl_8b_fsdp.sh 和 examples/grpo_trainer/run_qwen3_vl_8b_megatron.sh，README 的模型矩阵也列出 Qwen3-VL-8B 支持 FSDP/Megatron。你这里使用的是 v0.6.1，并且是把 Qwen2.5-VL 脚本替换成 Qwen3-VL 模型后运行；从现有信息看，还不足以判断高位抖动是框架 bug、环境版本问题、超参/数据问题，还是 Qwen3-VL 训练稳定性问题。
> 
> 为了继续定位，请补充：1) 完整启动命令和所有 Hydra override；2) verl commit、transformers/vllm/sglang/torch 版本；3) GPU/NPU 型号、卡数、FSDP 配置；4) wandb/console 中 reward、entropy、kl、grad_norm、clip_ratio 的曲线或关键日志；5) 是否能用当前 main 的 examples/grpo_trainer/run_qwen3_vl_8b_fsdp.sh 在 Geo3K 或最小数据上复现。
> 
> Thanks for the report. The current main branch has official Qwen3-VL-8B GRPO example scripts: examples/grpo_trainer/run_qwen3_vl_8b_fsdp.sh and examples/grpo_trainer/run_qwen3_vl_8b_megatron.sh, and the README support matrix lists Qwen3-VL-8B with FSDP/Megatron support. In this issue, the run uses v0.6.1 and a Qwen2.5-VL script with only the model swapped, so the available information is not enough to tell whether the oscillation is caused by a framework bug, dependency/version mismatch, hyperparameters/data, or Qwen3-VL training instability.
> 
> Please share: 1) the full launch command and Hydra overrides; 2) verl commit plus transformers/vllm/sglang/torch versions; 3) hardware and FSDP settings; 4) reward/entropy/kl/grad_norm/clip_ratio logs or curves; 5) whether current main's examples/grpo_trainer/run_qwen3_vl_8b_fsdp.sh reproduces the issue on Geo3K or a minimal dataset.
> 
> This response was generated by a scheduled Codex GPT5.5 issue processing task.

好像就是在脚本提前设置了两个参数 为了不让有warning信息设置了两个bfloat=16的配置导致的
一个是actor model 一个是ref model
